crates/bevy_scene/src/dynamic_scene.rs

```diff
@@ -35,7 +36,28 @@ impl DynamicScene {
         let mut scene = DynamicScene::default();
         let type_registry = type_registry.read();
 
-        for archetype in world.archetypes().iter() {
+        let mut archetypes = world.archetypes().iter();
+
+        // Empty archetype
+        let _ = archetypes.next().unwrap();
+
+        // Resources archetype
+        let resources_archetype = archetypes.next().unwrap();
```

Comment:
Ugh, I don't like that this is the best way we have to get this...

Reply:
Could we add a constant or dedicated method to fetch this?

Reply:
There is one, I'll switch to using it separately from the loop. The first two iterations of the loop will do nothing since there are no entities in the empty and resource archetypes.

Reply:
Fixed in f43d667